specs/signature-specification.md

```diff
@@ -64,13 +64,13 @@ Besides the [image manifest property requirements][image-manifest-property-descr
 
 The client should be able to discover all the signatures belonging to an artifact (such as image manifest) by using [OCI Distribution Referrers API][oci-distribution-referrers].
 OCI Distribution Referrers API returns a paginated list of all artifacts belonging to a target artifact (such as container images, SBoMs).
-The implementation can filter the Notary Project signatures by either using OCI Distribution Referrers API or using custom logic on the client.
-Each the Notary Project signature refers to a signature envelope blob.
+The implementation can filter the Notary signatures by either using OCI Distribution Referrers API or using custom logic on the client.
```

Comment:
We should replace "Notary signature" to "Notary Project signature" in all references in the doc.